apps/script/checkEntryPoints.js

```diff
@@ -84,7 +78,7 @@ const SILENCED = [
   'netsim',
   'pd',
   'plc',
-  'publicKeyCryptography',
+  'projectsList',
```

Comment:
I could have sworn it was a confusing error message the first time but when I look again it seems pretty clear:
```
✘ projectsList ➜ ./src/sites/studio/pages/projectsList.js
  ✘ Entry point names should match the name of the file they are used in.
    This entry point should be renamed to projects_lists/index!
  this entry point is referenced by the following templates:
    ../dashboard/app/views/projects_lists/index.html.haml
```